pkg/kubectl/cmd/util/helpers.go

```diff
@@ -725,3 +725,13 @@ func RequireNoArguments(c *cobra.Command, args []string) {
 		CheckErr(UsageError(c, fmt.Sprintf(`unknown command %q`, strings.Join(args, " "))))
 	}
 }
+
+// RunGetServerSupportedSMPatchVersion is a wrapper of GetServerSupportedSMPatchVersion(),
```

Comment:
Comment must start with "GetServerSupportedSMPatchVersionFromFactory" for godoc.